js/stringProcessing/cleanText.js

```diff
@@ -0,0 +1,53 @@
+/** @module stringProcessing/cleanText */
```

Comment:
Consider if this is something that should be exposed as a separate API, maybe we should just move this to the fleschReadingEasy.js file.